### PR TITLE
CI: Fix working directories to be relative

### PR DESCRIPTION
**Pull request by danmcp**
This should get PR validation back to a working state.  Unfortunately it has to merge before it will take effect.

### CODE REVIEW

File: .github/workflows/e2e-nvidia-t4-x1.yml [hunk 1]
```diff
@@ -86,7 +86,7 @@ jobs:
           fetch-depth: 0
 
       # for debugging
-      - name: Print environment state
+      - name: Print Environment State
```
Comment (nathan-weinberg) [inline .github/workflows/e2e-nvidia-t4-x1.yml:89]: I'd prefer to keep this as is

File: .github/workflows/e2e-nvidia-t4-x1.yml [hunk 2]
```diff
@@ -95,17 +95,18 @@ jobs:
       - name: Fetch and checkout PR
         id: fetch_pr
         if: github.event_name == 'pull_request_target'
+        working-directory: ./eval
         run: |
           git fetch origin pull/${{ github.event.pull_request.number }}/head:pr-${{ github.event.pull_request.number }}
           git checkout pr-${{ github.event.pull_request.number }}
 
-      - name: Install system packages
+      - name: Install Packages
```
Comment (nathan-weinberg) [inline .github/workflows/e2e-nvidia-t4-x1.yml:103]: I'd prefer to keep this as is

**Review by nathan-weinberg — COMMENTED** [review]
two nits but otherwise LGTM

File: .github/workflows/e2e-nvidia-t4-x1.yml (hunk 2 above)
Comment (danmcp) [reply .github/workflows/e2e-nvidia-t4-x1.yml]: I was trying to match the state of the training repo and missed.  Fixed!

**Comment by nathan-weinberg** [comment]
Merging manually since CI needs this fix to pass

**Pull request merged**